openssh/Patches/0019-PubKeyPlugin-support.patch

```diff
@@ -201,35 +138,35 @@ index 5d59febc..18a93889 100644
  	    remote_host, conn_id, rdomain, &opts)) != 0)
  		goto out;
 diff --git a/servconf.c b/servconf.c
-index f8b14368..e4259285 100644
+index ba3b16fd6..69f5ac0b8 100644
 --- a/servconf.c
 +++ b/servconf.c
-@@ -211,6 +211,7 @@ initialize_server_options(ServerOptions *options)
+@@ -209,6 +209,7 @@ initialize_server_options(ServerOptions *options)
  	 */
  	options->pam_service_per_authmethod = 1;
  #endif
 +	options->pubkey_plugin = NULL;
  }
  
  /* Returns 1 if a string option is unset or set to "none" or 0 otherwise. */
-@@ -566,7 +567,7 @@ typedef enum {
- 	sStreamLocalBindMask, sStreamLocalBindUnlink,
+@@ -565,7 +566,7 @@ typedef enum {
  	sAllowStreamLocalForwarding, sFingerprintHash, sDisableForwarding,
  	sExposeAuthInfo, sRDomain, sPubkeyAuthOptions, sSecurityKeyProvider,
  	sRequiredRSASize, sChannelTimeout, sUnusedConnectionTimeout,
 -	sDeprecated, sIgnore, sUnsupported
 +	sPubKeyPlugin, sDeprecated, sIgnore, sUnsupported
  } ServerOpCodes;
  
-@@ -766,6 +767,7 @@ static struct {
+ #define SSHCFG_GLOBAL		0x01	/* allowed in main section of config */
```

Comment:
Why is this here?

Reply:
I think this could be the difference in git versions. I didn't alter `serverconf.c` but I did regenerate the changes by applying the original with gpatch, making my changes, then created this file via `git format-patch`.

Let me know if you'd prefer me to just take everything for `serverconf.c` or `servconf.h` from the original patch. I've not modified anything in those files so I can manually edit the patch and retest.

Reply:
I was expecting there to be no *additions*.  If it builds I guess(?) It's okay, but I can imagine maybe gettings a double-define error in compilation.

Reply:
DOH!  Upon looking at the raw diff file... I see I was viewing the + sign in the wrong column.